### DISCUSSION TITLE
inject node is not working as expected

### DISCUSSION THREAD

**Discussion by soesas** [zachowj/hass-node-red#321] [Q&A] [answered]
unfortunately the inject node is not working as expected. The mouse symbol does not change and on click nothing happens. 
However when I click inside the inject node it is working.

**Comment by soesas** [comment]
![help](https://github.com/user-attachments/assets/b6160d0e-7f77-4b37-a6d2-c6d9a87f619b)
![help_working](https://github.com/user-attachments/assets/a8956103-b5fc-4495-88c4-1152833c21fc)


Maybe it helps to unterstand what I mean. 

Thanks for your help.

**Reply by zachowj** [reply, marked as answer]
Are you using Node-RED as a Home Assistant add-on? If so, try accessing the Node-RED UI directly on port 1880. If this resolves your issue, ensure you're running the latest version of this package to enable the use of inject nodes via the Home Assistant ingress proxy.  

For more details on the root cause of this issue, see: https://github.com/hassio-addons/addon-node-red/issues/1986.

**Comment by soesas** [comment]
Awesome.
Re adding the Add On did the trick I don't exactly know why but I'm happy that it's working now.
I honestly forgot about the ad on usually I do things via H A C S or directly in the integration and not the add-on. so sorry for that. Thank you very much
